frontend/src/components/pages/DeletionJob.js

```diff
@@ -149,6 +146,8 @@ const DeletionJob = ({ gateway, jobId }) => {
     setExportState("initial");
   };
 
+  const isLegacy = (job) => !isUndefined(job.DeletionQueueItems);
```

Comment:
Perhaps we can find a better term than `isLegacy`? I'm thinking what if we deprecate a different field in a future release that has no relation to the presence of the `DeletionQueueItems`

Reply:
I added a little comment and renamed, better?